tar/write.c

```diff
@@ -1083,7 +1083,8 @@ write_entry_backend(struct bsdtar *bsdtar, struct archive *a,
 	 */
 	if ((archive_entry_size(entry) > 0) && (filecached == 0)) {
 		const char *pathname = archive_entry_sourcepath(entry);
-		fd = open(pathname, O_RDONLY);
+		fd = fileutil_open_noatime(pathname, O_RDONLY,
```

Comment:
How are we going to call this if we don't `#include "fileutil.h"`?

Reply:
BTW, this 1-line change is all that's needed to open files with `noatime`.  (Plus the boilerplate of adding the `--noatime` option, etc.)

Opening directories is, relatively speaking, more complicated.  That's why I was musing about splitting this into two separate PRs: one for files, one for directories.

Reply:
Yes, that #include should be in this ("Optionally open files with O_NOATIME") commit, not the "Add --noatime boilerplate" commit.

Reply:
Oops, fixed.